templates/default/groups.css

```diff
@@ -1,11 +1,42 @@
-.xoct_add_user, .xoct_remove_user {
-    position: absolute;
-    top: 6px;
-    right: 5px;
-    padding: 5px 8px !important;
+#xoct_available_participant, #xoct_group_participant {
+    padding: 0;
+    margin: 0;
 }
 
-.xoct_checkmark {
-    float: left;
-    margin: 5px;
-}
\ No newline at end of file
+.xoct_participant_available, li.list-group-item, a.list-group-item {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 0.5rem;
+    border-bottom: 1px solid #ddd;
+    padding: 0.5rem 1rem;
+}
+
+.list-group-item > div, .list-group-item > span {
+    flex-grow: 1;
+    margin: 0;
+}
+
+.list-group-item button {
+    flex-shrink: 0;
+}
+
+.xoct_add_user {
+    margin-left: auto;
+    margin-right: 0;
+    background-color: #007bff;
+}
+
+.btn-default.xoct_group_delete {
+    background-color: #f63b4c;
+    border-color: #dc3545;
+}
+
```

Comment:
Unwanted new line! Please remove it and keep it at max 1 new line so that it would look better.